lms/static/sass/discussion/views/_create-edit-post.scss

```diff
@@ -9,7 +9,6 @@
   margin: 0;
   border-radius: 3px;
   padding: ($baseline*2);
-  min-width: 760px;
```

Comment:
Context: This fixes a visual regression / issue where the edit state of responses and comments is visually misaligned.

Screenshot:
![image](https://cloud.githubusercontent.com/assets/2023680/9752013/bbafa330-5679-11e5-8a0e-db3b55eb2d90.png)